### ISSUE TITLE
[google_sign_in] iOS build failure on m1 mac on upgrading to stable 2.8

### ISSUE THREAD

**Issue by Alverd04** [flutter/flutter#95206]
logs: 
```
  ** BUILD FAILED **
           
           
[        ] Xcode's output:
           ↳
[        ]     Command line invocation:
                   /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -configuration Debug
                   VERBOSE_SCRIPT_LOGGING=YES -workspace Runner.xcworkspace -scheme Runner
                   BUILD_DIR=/Users/albertfont/my-event-app/build/ios -sdk iphonesimulator -destination
                   id=FC4BC446-CD0F-404A-9BB1-AF06126C4DCE

SCRIPT_OUTPUT_STREAM_FILE=/var/folders/c0/cvbsqsy16kvgkkbq500w4ts80000gn/T/flutter_tools.ku5YyS/fl
                   utter_build_log_pipe.Ia30k3/pipe_to_stdout FLUTTER_SUPPRESS_ANALYTICS=true
                   COMPILER_INDEX_STORE_ENABLE=NO
           
               User defaults from command line:
                   IDEPackageSupportUseBuiltinSCM = YES
           
               Build settings from command line:
                   BUILD_DIR = /Users/albertfont/my-event-app/build/ios
                   COMPILER_INDEX_STORE_ENABLE = NO
                   FLUTTER_SUPPRESS_ANALYTICS = true
                   SCRIPT_OUTPUT_STREAM_FILE =

/var/folders/c0/cvbsqsy16kvgkkbq500w4ts80000gn/T/flutter_tools.ku5YyS/flutter_build_log_pipe.Ia30k
                   3/pipe_to_stdout
                   SDKROOT = iphonesimulator15.0
                   VERBOSE_SCRIPT_LOGGING = YES
           
               note: Using new build system
               note: Planning
               Analyze workspace
           
               Create build description
               Build description signature: 7014f3ca6bf128ecc20d9665e8256072
               Build description path:

/Users/albertfont/Library/Developer/Xcode/DerivedData/Runner-arsafayndnmhrveorroeorekeuos/Build/Interm
               ediates.noindex/XCBuildData/7014f3ca6bf128ecc20d9665e8256072-desc.xcbuild
           
               note: Build preparation complete
               note: Building targets in parallel
               /Users/albertfont/my-event-app/ios/Runner.xcodeproj: error: The linked framework
               'Pods_Runner.framework' is missing one or more architectures required by this target:
x86_64. (in
               target 'Runner' from project 'Runner')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator
deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
versions is
               9.0 to 15.0.99. (in target 'AppAuth' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator
deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
versions is
               9.0 to 15.0.99. (in target 'GoogleSignIn' from project 'Pods')
```

**Comment by mikaryyn** [comment]
I think this is the same issue as here, https://github.com/flutter/flutter/issues/94914#issuecomment-992898782. Please try the described Podfile change as workaround.

**Comment by Alverd04** [comment]
Still does not work

**Comment by Ulukbek-01** [comment]
Same here as well. After adding google_sign_in package, my app has stopped working and showing the problem above. Has anyone solved this issue?

**Comment by maheshmnj** [comment]
Hi @Alverd04, Thanks for filing the issue. I am able to reproduce the issue. The build fails after adding the `google_sign_in` plugin on stable and the master channel. 

>  error: The linked framework 'Pods_Runner.framework' is missing one or more architectures required by this target: x86_64. (in target 'Runner' from project 'Runner')

complete [flutter run -v](https://github.com/flutter/flutter/files/7710184/log.txt) logs. 
 
On a side note even after setting the deployment target to 10 via xcode in the build errors shows the warning

> warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'GoogleSignIn' from project 'Pods')
    /Users/mahesh/Desktop/hello/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'AppAuth' from project 'Pods')
 
<details>
<summary>pubspec.yaml</summary>

```yaml
name: hello
description: A new Flutter project.

# The following line prevents the package from being accidentally published to
# pub.dev using `flutter pub publish`. This is preferred for private packages.
publish_to: 'none' # Remove this line if you wish to publish to pub.dev

# The following defines the version and build number for your application.
# A version number is three numbers separated by dots, like 1.2.43
# followed by an optional build number separated by a +.
# Both the version and the builder number may be overridden in flutter
# build by specifying --build-name and --build-number, respectively.
# In Android, build-name is used as versionName while build-number used as versionCode.
# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
# Read more about iOS versioning at
# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
version: 1.0.0+1

environment:
  sdk: ">=2.12.0 <3.0.0"

# Dependencies specify other packages that your package needs in order to work.
# To automatically upgrade your package dependencies to the latest versions
# consider running `flutter pub upgrade --major-versions`. Alternatively,
# dependencies can be manually updated by changing the version numbers below to
# the latest version available on pub.dev. To see which dependencies have newer
# versions available, run `flutter pub outdated`.
dependencies:
  flutter:
    sdk: flutter

  # The following adds the Cupertino Icons font to your application.
  # Use with the CupertinoIcons class for iOS style icons.
  cupertino_icons: ^1.0.2
  google_sign_in: ^5.2.1

dev_dependencies:
  flutter_test:
    sdk: flutter

  # The "flutter_lints" package below contains a set of recommended lints to
  # encourage good coding practices. The lint set provided by the package is
  # activated in the `analysis_options.yaml` file located at the root of your
  # package. See that file for information about deactivating specific lint
  # rules and activating additional ones.
  flutter_lints: ^1.0.0

# For information on the generic Dart part of this file, see the
# following page: https://dart.dev/tools/pub/pubspec

# The following section is specific to Flutter.
flutter:

  # The following line ensures that the Material Icons font is
  # included with your application, so that you can use the icons in
  # the material Icons class.
  uses-material-design: true

  # To add assets to your application, add an assets section, like this:
  # assets:
  #   - images/a_dot_burr.jpeg
  #   - images/a_dot_ham.jpeg

  # An image asset can refer to one or more resolution-specific "variants", see
  # https://flutter.dev/assets-and-images/#resolution-aware.

  # For details regarding adding assets from package dependencies, see
  # https://flutter.dev/assets-and-images/#from-packages

  # To add custom fonts to your application, add a fonts section here,
  # in this "flutter" section. Each entry in this list should have a
  # "family" key with the font family name, and a "fonts" key with a
  # list giving the asset and other descriptors for the font. For
  # example:
  # fonts:
  #   - family: Schyler
  #     fonts:
  #       - asset: fonts/Schyler-Regular.ttf
  #       - asset: fonts/Schyler-Italic.ttf
  #         style: italic
  #   - family: Trajan Pro
  #     fonts:
  #       - asset: fonts/TrajanPro.ttf
  #       - asset: fonts/TrajanPro_Bold.ttf
  #         weight: 700
  #
  # For details regarding fonts from package dependencies,
  # see https://flutter.dev/custom-fonts/#from-packages

```
</details>

<details>
<summary>Podfile</summary>

```ruby
# Uncomment this line to define a global platform for your project
# platform :ios, '9.0'

# CocoaPods analytics sends network stats synchronously affecting flutter build latency.
ENV['COCOAPODS_DISABLE_STATS'] = 'true'

project 'Runner', {
  'Debug' => :debug,
  'Profile' => :release,
  'Release' => :release,
}

def flutter_root
  generated_xcode_build_settings_path = File.expand_path(File.join('..', 'Flutter', 'Generated.xcconfig'), __FILE__)
  unless File.exist?(generated_xcode_build_settings_path)
    raise "#{generated_xcode_build_settings_path} must exist. If you're running pod install manually, make sure flutter pub get is executed first"
  end

  File.foreach(generated_xcode_build_settings_path) do |line|
    matches = line.match(/FLUTTER_ROOT\=(.*)/)
    return matches[1].strip if matches
  end
  raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
end

require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)

flutter_ios_podfile_setup

target 'Runner' do
  use_frameworks!
  use_modular_headers!

  flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
  end
end

```
</details>

 
 <details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 2.8.0, on macOS 12.0.1 21A559 darwin-arm, locale en-GB)
    • Flutter version 2.8.0 at /Users/mahesh/Documents/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision cf44000065 (5 days ago), 2021-12-08 14:06:50 -0800
    • Engine revision 40a99c5951
    • Dart version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/mahesh/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • ANDROID_HOME = /Users/mahesh/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.10.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] IntelliJ IDEA Community Edition (version 2021.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 60.1.4
    • Dart plugin version 212.5080.8

[✓] VS Code (version 1.61.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.29.0

[✓] Connected device (3 available)
    • iPhone 11 (mobile) • FC644E6C-7F02-40E5-9998-1F0CD162FBD8 • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • macOS (desktop)    • macos                                • darwin-arm64   • macOS 12.0.1 21A559 darwin-arm
    • Chrome (web)       • chrome                               • web-javascript • Google Chrome 96.0.4664.93

• No issues found!
```

```
[✓] Flutter (Channel master, 2.6.0-12.0.pre.1002, on macOS 12.0.1 21A559 darwin-arm, locale en-GB)
    • Flutter version 2.6.0-12.0.pre.1002 at /Users/mahesh/Documents/flutter_master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 9de8849905 (31 hours ago), 2021-12-12 20:54:15 -0500
    • Engine revision f16c96c31f
    • Dart version 2.16.0 (build 2.16.0-85.0.dev)
    • DevTools version 2.9.1

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/mahesh/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • ANDROID_HOME = /Users/mahesh/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.10.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] IntelliJ IDEA Community Edition (version 2021.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 60.1.4
    • Dart plugin version 212.5080.8

[✓] VS Code (version 1.61.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.29.0

[✓] Connected device (3 available)
    • iPhone 11 (mobile) • FC644E6C-7F02-40E5-9998-1F0CD162FBD8 • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • macOS (desktop)    • macos                                • darwin-arm64   • macOS 12.0.1 21A559 darwin-arm
    • Chrome (web)       • chrome                               • web-javascript • Google Chrome 96.0.4664.93

• No issues found!
```
</details>

**Comment by Alverd04** [comment]
Logs are the following:
```
[  +29 ms] Running Xcode build... (completed in 5,2s)
[   +2 ms] Xcode build done.                                            5,2s
[   +7 ms] Failed to build iOS app
[        ] Error output from Xcode build:
           ↳
[   +1 ms]     --- xcodebuild: WARNING: Using the first of multiple matching destinations:
               { platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder,
name:Any iOS
               Simulator Device }
               { platform:iOS Simulator, id:04DB7655-53B8-4F32-8F62-0843E3A2D195, OS:15.0, name:iPad (9th generation) }
               { platform:iOS Simulator, id:FA886554-53FF-486D-9910-65CFFD8B735B, OS:15.0, name:iPad Air (4th generation)
}
               { platform:iOS Simulator, id:D5FEC67E-94FD-4D81-82C4-E8E138A4077C, OS:15.0, name:iPad Pro (9.7-inch) }
               { platform:iOS Simulator, id:D7264E89-E7BE-434A-9FC8-C6C22A9BEB02, OS:15.0, name:iPad Pro (11-inch) (3rd
generation) }
               { platform:iOS Simulator, id:20F62C0F-52C7-4346-8735-90279E29171F, OS:15.0, name:iPad Pro (12.9-inch) (5th
generation)
               }
               { platform:iOS Simulator, id:18F9520A-E83A-4BDC-87EC-4F02248C197E, OS:15.0, name:iPad mini (6th generation)
}
               { platform:iOS Simulator, id:E12B88A4-E949-4FD8-9B69-37F4E88B397F, OS:15.0, name:iPhone 8 }
               { platform:iOS Simulator, id:827A7AB0-E101-42F0-99D1-7E7F70BBB91F, OS:15.0, name:iPhone 8 Plus }
               { platform:iOS Simulator, id:BF26D24A-8D54-4EB7-81B9-CFC5BD0C53FB, OS:15.0, name:iPhone 11 }
               { platform:iOS Simulator, id:ADC0CE04-5AC9-431E-B6E3-7B83D0297E30, OS:15.0, name:iPhone 11 Pro }
               { platform:iOS Simulator, id:9BD9179C-F02B-41D3-9AE2-9FE881C30C8D, OS:15.0, name:iPhone 11 Pro Max }
               { platform:iOS Simulator, id:09CCF91A-8B65-4B90-9909-4D6606BF9AE8, OS:15.0, name:iPhone 12 }
               { platform:iOS Simulator, id:A377544B-8825-4E81-9E89-36317EB80285, OS:15.0, name:iPhone 12 Pro }
               { platform:iOS Simulator, id:D27107B9-5BD1-436D-BA7C-28BE1D929D60, OS:15.0, name:iPhone 12 Pro Max }
               { platform:iOS Simulator, id:B33D5C69-F088-4DCB-A916-85181DD6A2ED, OS:15.0, name:iPhone 12 mini }
               { platform:iOS Simulator, id:FC4BC446-CD0F-404A-9BB1-AF06126C4DCE, OS:15.0, name:iPhone 13 }
               { platform:iOS Simulator, id:E35FE895-2C89-4666-A1C5-AC11A490F3C9, OS:15.0, name:iPhone 13 Pro }
               { platform:iOS Simulator, id:056DD50D-734B-4EF1-8B13-BCD5540C09ED, OS:15.0, name:iPhone 13 Pro Max }
               { platform:iOS Simulator, id:F345104D-6D18-4268-933B-095AD8871074, OS:15.0, name:iPhone 13 mini }
               { platform:iOS Simulator, id:3C5DE004-D664-4E0B-B672-9F5749BE4FFE, OS:15.0, name:iPhone SE (2nd generation)
}
               { platform:iOS Simulator, id:FA9D210B-4FF4-4C2E-A209-3EB9810E0A14, OS:15.0, name:iPod touch (7th
generation) }
               { platform:macOS, arch:arm64, variant:Designed for [iPad,iPhone], id:00008103-001551390EDA001E }
               { platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
               ** BUILD FAILED **
           
           
[   +2 ms] Xcode's output:
           ↳
[   +3 ms]     Command line invocation:
                   /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -configuration Debug
VERBOSE_SCRIPT_LOGGING=YES
                   -workspace Runner.xcworkspace -scheme Runner BUILD_DIR=/Users/albertfont/my-event-app/build/ios -sdk
                   iphonesimulator

SCRIPT_OUTPUT_STREAM_FILE=/var/folders/c0/cvbsqsy16kvgkkbq500w4ts80000gn/T/flutter_tools.0d2bDr/flutter_build_log_
                   pipe.IkMpbf/pipe_to_stdout FLUTTER_SUPPRESS_ANALYTICS=true COMPILER_INDEX_STORE_ENABLE=NO
           
               User defaults from command line:
                   IDEPackageSupportUseBuiltinSCM = YES
           
               Build settings from command line:
                   BUILD_DIR = /Users/albertfont/my-event-app/build/ios
                   COMPILER_INDEX_STORE_ENABLE = NO
                   FLUTTER_SUPPRESS_ANALYTICS = true
                   SCRIPT_OUTPUT_STREAM_FILE =

/var/folders/c0/cvbsqsy16kvgkkbq500w4ts80000gn/T/flutter_tools.0d2bDr/flutter_build_log_pipe.IkMpbf/pipe_to_stdout
                   SDKROOT = iphonesimulator15.0
                   VERBOSE_SCRIPT_LOGGING = YES
           
               note: Using new build system
               note: Planning
               Analyze workspace
           
               Create build description
               Build description signature: 1b0290a690c958b6fd6ad43b31405112
               Build description path:

/Users/albertfont/Library/Developer/Xcode/DerivedData/Runner-arsafayndnmhrveorroeorekeuos/Build/Intermediates.noindex/
               XCBuildData/1b0290a690c958b6fd6ad43b31405112-desc.xcbuild
           
               note: Build preparation complete
               note: Building targets in parallel
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'MTBBarcodeScanner' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Runner.xcodeproj: error: The linked framework 'Pods_Runner.framework' is
missing
               one or more architectures required by this target: x86_64. (in target 'Runner' from project 'Runner')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'gRPC-C++' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'GoogleSignIn' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'gRPC-C++-gRPCCertificates-Cpp' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'AppAuth' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'BoringSSL-GRPC' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'abseil' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'gRPC-Core' from project 'Pods')
               /Users/albertfont/my-event-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target
               'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
15.0.99.
               (in target 'leveldb-library' from project 'Pods')
           
[  +21 ms] Could not build the application for the simulator.
[  +11 ms] Error launching application on iPhone 13.
[  +15 ms] "flutter run" took 21.128ms.
[  +16 ms] 
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
           #1      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:669:9)
           <asynchronous suspension>
           #2      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1125:27)
           <asynchronous suspension>
           #3      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #4      CommandRunner.runCommand (package:args/command_runner.dart:209:13)
           <asynchronous suspension>
           #5      FlutterCommandRunner.runCommand.<anonymous closure>
           (package:flutter_tools/src/runner/flutter_command_runner.dart:288:9)
           <asynchronous suspension>
           #6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #7      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:236:5)
           <asynchronous suspension>
           #8      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:62:9)
           <asynchronous suspension>
           #9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #10     main (package:flutter_tools/executable.dart:92:3)
           <asynchronous suspension>
           
           
[ +144 ms] ensureAnalyticsSent: 138ms
[   +1 ms] Running shutdown hooks
[        ] Shutdown hooks complete
[        ] exiting with code 1
```

**Comment by MonsieurZbanowanYY** [comment]
same issue for me

**Comment by jmagman** [comment]
Duplicate of https://github.com/flutter/flutter/issues/94914.
